server/storage/data-model.ts

```diff
@@ -0,0 +1,23 @@
+type Message = {
+  id: string;
+  timestamp: number;
+  text: string;
+  senderId: string;
+};
+
+type User = {
+  id: string;
+  password: string;
+  name: string;
+  profilePicture: string;
+  chatRooms: string[];
+};
+
+type ChatRoom = {
+  id: string;
+  name?: string;
+  type: "dm" | "channel";
```

Comment:
Types in capital letter. "dm" -> "DM"

Reply:
This is room classification into dm or channel (not the Typescript types so that's why they are in lowercase)